keras_core/backend/torch/math.py

```diff
@@ -408,3 +408,9 @@ def istft(
 def rsqrt(x):
     x = convert_to_tensor(x)
     return torch.rsqrt(x)
+
+
+def erf(x):
+    if not isinstance(x, torch.Tensor):
+        x = torch.tensor(x)
```

Comment:
You can just do `x = convert_to_tensor(x)` unconditionally

Reply:
Done